opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -300,6 +346,43 @@ enum AggregationTemporality {
   AGGREGATION_TEMPORALITY_CUMULATIVE = 2;
 }
 
+// ScalarDataPoint is a single data point in a timeseries that describes the
+// time-varying value of a integer or double-valued metric.
+message ScalarDataPoint {
+  // The set of labels that uniquely identify this timeseries.
+  repeated opentelemetry.proto.common.v1.StringKeyValue labels = 1;
```

Comment:
Probably better as next review. Subbing out Labels.

Reply:
@victlu also mentioned this concern, and I for one will support a proposal that moves label sets into a new location of the protocol so they can be represented by integers referring to a dictionary entry at this level in the protocol.